test/botkubex/botkube_cli_helpers.go

```diff
@@ -49,13 +49,17 @@ func Install(t *testing.T, params InstallParams) error {
 		"--set-string",
 		fmt.Sprintf("extraEnv[1].value=%ds", params.PluginRestartHealthCheckIntervalSeconds),
 		"--set",
+		fmt.Sprintf("extraEnv[2].name=%s", "BOTKUBE_SETTINGS_UPGRADE_NOTIFIER"),
```

Comment:
That shouldn't be necessary as AFAIR we checked a few last messages to wait for the welcome + help messages. Anyway, ofc I'm okay with disabling it 👍